contrib/time_series_expectations/time_series_expectations/expectations/expect_column_pair_values_to_match_prophet_date_model.py

```diff
@@ -0,0 +1,211 @@
+from typing import Optional
+
+import pandas as pd
+
+from great_expectations.core.expectation_configuration import ExpectationConfiguration
+from great_expectations.data_context.util import file_relative_path
+from great_expectations.execution_engine import (
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnPairMapExpectation
+from great_expectations.expectations.metrics.import_manager import F, sparktypes
+from great_expectations.expectations.metrics.map_metric_provider import (
+    ColumnPairMapMetricProvider,
+    column_pair_condition_partial,
+)
+from time_series_expectations.expectations.util import get_prophet_model_from_json
```

Comment:
Could we instead add this to the package instead of in a utils file? I would prefer to see something like a `ProphetModelDeserializer` in something like `contrib/time_series_expectations/time_series_expectations/prophet_model_deserializer.py` that gets composed into expectations that use it.

Reply:
Done. Thanks for the guidance!

Reply:
Thanks for implementing! I do see where the ProphetModelDeserializer is used, but I don't see the definition of that class. Did I miss it in the diff?

Reply:
Oops. I hadn't committed the file. It's there now